compute/admin_guide/authentication/oidc.adoc

```diff
@@ -87,7 +87,6 @@ Enter the following callback URL:
 +
```

Comment:
- Line 3 - an standard -> a standard
    - Line 5 - Add Azure AD

Reply:
I've added a list of all providers with in-page links to make it easier for customers to jump to the instructions they need.